Assets/HoloToolkit/Input/Scripts/InputManager.cs

```diff
@@ -494,10 +494,10 @@ public void RaiseSourceLost(IInputSource source, uint sourceId, object tag = nul
                 handler.OnManipulationStarted(casted);
             };
 
-        public void RaiseManipulationStarted(IInputSource source, uint sourceId, Vector3 cumulativeDelta, object tag = null)
+        public void RaiseManipulationStarted(IInputSource source, uint sourceId, object tag = null)
         {
             // Create input event
-            manipulationEventData.Initialize(source, sourceId, tag, cumulativeDelta);
+            manipulationEventData.Initialize(source, sourceId, tag, Vector3.zero);
```

Comment:
I just downloaded b8 from Unity's site, and it will not build without these changes. What are you referring to?